builds/codex-v102-neurocybernetics/neuro/signal_models.py

```
@@ -0,0 +1,9 @@
+import numpy as np
+
+def bandpower(signal, fs=256.0):
```

Ruff (0.14.3)

Unused function argument: `fs`

(ARG001)